daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md

```diff
@@ -199,6 +199,21 @@ await foreach (var items in subscribeConfigurationResponse.Source.WithCancellati
 }
 ```
 
+### Manage workflow instances (Alpha)
+
+```csharp
+var daprClient = new DaprClientBuilder().Build();
+
+// Start workflow
```

Comment:
string instanceId;
string workflowComponentName;
string workflowName;
object input ;
Dictionary<string, string> workflowOptions; // This is an optional parameter
CancellationToken cts;

It might be helpful to add all of these above "//Start workflow" to give the user a better idea as to what variable types everything is.

Reply:
Don't bother with a `CancellationToken cts` local variable. Just pass `CancellationToken.None` directly as the last input of `StartWorkflowAsync`.

Reply:
If we're using `CancellationToken.None` I'd rather just use the default variable. I worry that this implies that you _have_ to use a `CancellationToken` instead of having it as optional.

Reply:
Oh, good point. In that case, let’s remove both the options variable and the cancellation token.